src/main/proto/ga4gh/variants.proto

```diff
@@ -191,4 +191,29 @@ message Variant {
   // determination of genotype with respect to this variant. `Call`s in this
   // array are implicitly associated with this `Variant`.
   repeated Call calls = 12;
+
+  // The "variant_type" is used to denote e.g. structural variants.
+  // Examples:
+  //   DUP  : duplication of sequence following "start"; not necessarily in situ
+  //   DEL  : deletion of sequence following "start"
+  string variant_type = 13;
```

Comment:
@mbaudis Great, sounds good to me! Variant callers want to be able to describe their own variant types and they use their own plain strings to do so. Giving the option of upgrading to a controlled vocabulary is something we might consider doing in other places in the API as well.

It is actually a very useful way to understand how the science moves forward: as new string values for variant type become popular, the sequence ontology has a strong case for reviewing adding it to their vocabulary. 

VCF spec has `SVTYPE` https://samtools.github.io/hts-specs/VCFv4.2.pdf using BND in the type. I believe these need to be interchanged as well. This post is very helpful on the topic of SV calls: https://simpsonlab.github.io/2015/06/15/merging-sv-calls/ .